### ISSUE TITLE
Automatically load TikTok base pixel

### ISSUE THREAD

**Issue by luukfiets** [tiktok/gtm-template-pixel#18]
Currently this tag template only checks whether the base tag is present but doesn't load the base tag in case it was not found. It would be really useful if the tiktok base pixel would be injected if it was not found, just like the Meta tag template does.

Please let me know if I can help with setting this up. I'm happy to suggest the required changes.

**Comment by tt-hubenlin** [comment]
Thank you for your suggestion. Currently our [GTM interactive setup flow](https://business-api.tiktok.com/portal/docs?id=1799004034003969) will help users install base-code tags and template tags. Could you provide the link to the tag template you mentioned? We will carefully evaluate the benefits and other impacts of providing this feature.

**Comment by luukfiets** [comment]
@tt-hubenlin, of course: on these lines the Facebook tag template checks whether the fbq library is already loaded: https://github.com/facebookarchive/GoogleTagManager-WebTemplate-For-FacebookPixel/blob/main/template.tpl#L548-L553.

If it's not loaded yet, it initiates the pixel (first loading consent status before actually injecting the tracking script) and then loads the actual script: https://github.com/facebookarchive/GoogleTagManager-WebTemplate-For-FacebookPixel/blob/main/template.tpl#L555-L619.

This setup makes it easier to set up the pixel for non-tech savvy users.

**Comment by prod1992** [comment]
This is a very good point, and would be really useful to have it.
